src/localize/languages/pt.json

```diff
@@ -0,0 +1,33 @@
+{
+  "common": {
+    "invalid_configuration": "Configuração inválida.",
+    "entity_missing": "A entidade de clima não foi encontrada.",
+    "invalid_forecast_days": "forecast_days deve ser maior do que 0."
+  },
+  "weather": {
+    "clear-night": "Céu limpo, sem nuvens",
+    "cloudy": "Nublado",
+    "fog": "Nevoeiro",
+    "hail": "Granizo",
+    "lightning": "Raios",
+    "lightning-rainy": "Chuvoso com raios",
+    "partlycloudy": "Parcialmente nublado",
+    "pouring": "Torrencial",
+    "rainy": "Chuvoso",
+    "snowy": "Neve",
+    "snowy-rainy": "Chuvoso com neve",
+    "sunny": "Ensolarado",
+    "windy": "Ventania",
+    "windy-variant": "Ventania forte",
+    "exceptional": "Excepcional"
+  },
+  "day": {
```

Comment:
can you abbreviate the days to 2-3 characters?
it would otherwise break the layout of the card 
<img width="442" alt="image" src="https://user-images.githubusercontent.com/33731393/196514585-76b85975-f269-4d9b-9030-6320f09bccfb.png">

Reply:
Sure! Done, thanks!!!

Reply:
Not sure I managed to do the new pull request, but the 3 letter abbreviation is simply the first 3 letters of each weekday, just cut out the rest, please! Thanks again!